README.md

```diff
@@ -0,0 +1,55 @@
+# eslint-config-feedzai
+
+This package provides the ESLint configuration that we use at Feedzai in order check our code conventions and coding style in JavaScript projects. The objective of this package is provide the ESLint configurations as an extensible shared configuration that could be used by everyone and across the Feedzai's products.
+
+This repo includes two distinguished ESLint configurations:
+
+- `eslint-config-feedzai`- This package is our default configs that contain all of our ESLint rules, including our rules for ECMAScript 6+. If you're using React for your projects please check the `eslint-config-feedzai-react`.
```

Comment:
"This package is our default configs that contain all of our ESLint rules..." I think it could be rephrased to
"This package contains our ESLint rules, including the ones for ECMAScript 6+."